### PR TITLE
Add missing Valdr typings for validation results and constraints

### PR DESCRIPTION
**Pull request by christophercr**
Add ValdrValidationResult and ValdrConstraints typings

### CODE REVIEW

**Comment by christophercr** [comment]
Hi @ilbertz, could you have a look on these changes to see whether they can be merged?

Thanks!

File: valdr-message/valdr-message.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿// Type definitions for valdr.js v1.1.5 (valdr-message)
+﻿// Type definitions for valdr.js v1.1.5+ (valdr-message)
```
Comment (mhegazy) [inline valdr-message/valdr-message.d.ts]: this version number is used by other tools, e.g. publisher for @types. so please keep this to numbers.

File: valdr/valdr.d.ts [hunk 2]
```diff
@@ -1,25 +1,72 @@
-﻿// Type definitions for valdr.js v1.1.5
+﻿// Type definitions for valdr.js v1.1.5+
```
Comment (mhegazy) [inline valdr/valdr.d.ts]: same here.

**Review by mhegazy — CHANGES_REQUESTED** [review]

**Comment by christophercr** [comment]
Sorry for that. I've changed it.

Thanks for your review.

**Review by ilbertz — APPROVED** [review]
Looks good.

**Comment by mhegazy** [comment]
thanks @ilbertz for the review.

**Pull request merged**